pkg/reconciler/ingress/resources/httproute_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"knative.dev/networking/pkg/apis/networking/v1alpha1"
 	"knative.dev/pkg/kmeta"
 	"knative.dev/pkg/reconciler"
-	gwv1alpha1 "sigs.k8s.io/gateway-api/apis/v1alpha1"
+	gwv1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
```

Comment:
s/gwv1alpha2/gatewayv1alpha2 for consistency please. 


I think `gatewayv1alpha2` is a bettter alias name :)


Mind doing a search and replace for all files?